certify-service/src/main/java/io/mosip/certify/services/VCIssuanceServiceImpl.java

```diff
@@ -117,10 +117,12 @@ public Map<String, Object> getCredentialIssuerMetadata(String version) {
         if(issuerMetadata.containsKey(version)) {
             return issuerMetadata.get(version);
         } else if(version != null && version.equals("vd12")) {
-            Map<String, Object> vd12IssuerMetadata = convertLatestToVd12(issuerMetadata.get(version));
+            LinkedHashMap<String, Object> originalIssuerMetadata = new LinkedHashMap<>(issuerMetadata.get("latest"));
+            Map<String, Object> vd12IssuerMetadata = convertLatestToVd12(originalIssuerMetadata);
```

Comment:
At the end, can you add it to issuerMetadata class object, so we don't do the conversion everytime?

Reply:
Added to original issuer metadata.